CRM/Core/BAO/FinancialTrxn.php

```diff
@@ -555,34 +555,72 @@ public static function getPartialPaymentTrxn($contribution, $params) {
    *
```

Comment:
Please add to description that the membership revenue amount returned is going to be the amount to be recognized as of the $receiveDate for accrual accounting purposes